main.go

```diff
@@ -34,9 +35,25 @@ func main() {
 	if _, err = CreateAccount(ctx, client); err != nil {
 		log.Fatal(err)
 	}
-	if _, err = QueryAccount(ctx, client); err != nil {
+	a, err := QueryAccount(ctx, client)
+	if err != nil {
+		log.Fatal(err)
+	}
+	if _, err = CreateUser(ctx, client); err != nil {
+		log.Fatal(err)
+	}
+	u, err := QueryUser(ctx, client)
+	if err != nil {
 		log.Fatal(err)
 	}
+	if err := connectAccountToUser(ctx, client, a, u); err != nil {
+		log.Fatal(err)
+	}
+	log.Println("account connected to owner")
+}
+
+func connectAccountToUser(ctx context.Context, client *ent.Client, a *ent.Account, u *ent.User) error {
```

Comment:
Why do you need a function here? You can inline it, can't you?

Reply:
right, removed